src/test/java/sentinel/task/ToDoTest.java

```diff
@@ -0,0 +1,67 @@
+package sentinel.task;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class ToDoTest {
+
+    @Test
+    public void testConstructor() {
+        String description = "Test ToDo";
```

Comment:
You could consider having ``ToDo todo = new ToDo("Test ToDo");` declared at the class level so you won't have to repeat it for every JUnit test that utilizes it.

Reply:
Oh thank you for your suggestion. I was actually hesitant to do that due to the modifications I might make in each new test (eg: First test I just want to check on markDone and second test I want to test markUndone (for newly initialised ToDo instance that is already marked undone).